app/views/hq/users/show.html.erb

```diff
@@ -10,7 +10,8 @@
   <%= pluralize(@user.active_sponsors.count.to_s, 'Active Sponsor') %>
   <% if @user.sponsors.count > 0 %>
```

Comment:
I had specifically changed this from that; currently either both partials render or neither, but not just one.  This is a UI decision for the group